packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { cx } from '@leafygreen-ui/emotion';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import Tooltip from '@leafygreen-ui/tooltip';
+
+import { iconBaseStyles, iconThemeStyles } from './InfoSprinkle.styles';
+import { Align, InfoSprinkleProps, Justify } from './InfoSprinkle.types';
+
+export const InfoSprinkle = React.forwardRef<
+  HTMLSpanElement,
+  InfoSprinkleProps
+>(
+  (
+    {
+      darkMode: darkModeProp,
+      children,
+      baseFontSize,
+      ...rest
+    }: InfoSprinkleProps,
+    forwardRef,
+  ) => {
+    const { darkMode, theme } = useDarkMode(darkModeProp);
+    return (
+      <Tooltip
+        darkMode={darkMode}
+        baseFontSize={baseFontSize}
+        trigger={
+          <span
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+            tabIndex={0}
```

Comment:
I believe this should have a `role` attribute (I think `role="button"`)

Reply:
Should also have `aria-describedby=<tooltip-id>` (unless tooltip does the described by automatically)

Reply:
tooltip adds `aria-describedby`.

Reply:
Would having `role=button` imply that it's clickable?

Reply:
Re: zoom sync, we'll use `role="button"` and `aria-disabled` as done here https://cauldron.dequelabs.com/components/TooltipTabstop#help-icon